dbms/src/Common/Exception.h

```diff
@@ -73,6 +75,7 @@ using Exceptions = std::vector<std::exception_ptr>;
 
 std::string errnoToString(int code, int the_errno = errno);
 [[noreturn]] void throwFromErrno(const std::string & s, int code, int the_errno = errno);
+[[noreturn]] void throwFromErrno(const std::string & s, const std::string & path, int code, int the_errno = errno);
```

Comment:
We can use different name for this method for clarity. For example, `throwFromErrnoWithPath`.